src/content/docs/accounts/accounts-billing/new-relic-one-user-management/account-user-mgmt-tutorial.mdx

```diff
@@ -146,21 +146,23 @@ You can manage groups via either the UI or via API:
 
 Before adding access to groups, ensure you have a good understanding of the [user management concepts](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#understand-concepts) and have a good plan for the access you want to add. It may help to map out the groups, roles, and accounts you want users to have access to in a spreadsheet beforehand: here's an [example spreadsheet for planning access](https://docs.google.com/spreadsheets/d/1FnguDXRUX9FGY14oV4Gx6O08v4vNC2Pv0GGCsU7Pxuw/edit?usp=sharing). 
 
-To add access to groups in the UI:
+To create access grants from the UI:
 
-1. From the **Access management** UI, click **Groups**. 
-2. You can create a new group or edit an existing one: 
-  * To create a new group, click **Create new group** and follow the instructions. 
-  * To edit an existing one, click the `...` settings icon for the group you want to edit. Click **View group details** and edit the group's access. 
-
-<img title="New Relic user mgmt - create group" alt="New Relic user mgmt - create group" src={createGroupUi}/>
-<figcaption>On the **Groups** tab, you can click **Create new group** to add a user group.</figcaption>
-
-When you're done, if users are already assigned to the group you created or edited, those users will have the new access within a few minutes (although for [EU region organizations](/docs/using-new-relic/welcome-new-relic/get-started/our-eu-us-region-data-centers), this can take up to twenty minutes or so). If your users are not yet in that group (which would be true if you just created a new group), the next step will show you how to add users. 
+1. From the **Access management** UI, click **Access grants and groups**. 
+2. Choose one of the following:
+* **Existing**: If there is already a group you want to add an access grant to, you can use this. For example, if you want to gives users in the default [**Admin** or **User** group](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#groups) access to new accounts, you might choose this and then select the **Admin Default** or **Admin User** role.
```

Comment:
@zuluecho9 I would add four spaces to all child-level bullets to get more indentation (or two, two-space tabs). See our [style guide](https://docs.newrelic.com/docs/style-guide/structure/styleguide-markup-indentation#second-and-third) for the nitty gritty.

Reply:
oh right. i pasted this from raw version and forgot the indents. thanks